src/core/BUILD

```diff
@@ -1087,6 +1087,7 @@ grpc_cc_library(
         "absl/hash",
         "absl/strings",
     ],
+    visibility = ["@grpc:alt_grpc_base_legacy"],
```

Comment:
why is this visibility field needed?

Reply:
We want to reserve compatibility. The `grpc_base` (the target we want to remove from) has this visibility. And I believe this is mostly used for internal.